RecoTracker/MkFit/plugins/MkFitEventOfHitsProducer.cc

```diff
@@ -54,13 +55,15 @@ MkFitEventOfHitsProducer::MkFitEventOfHitsProducer(edm::ParameterSet const& iCon
       mkFitGeomToken_{esConsumes()},
       qualityToken_{esConsumes()},
       geomToken_{esConsumes()},
-      putToken_{produces<MkFitEventOfHits>()} {}
+      putToken_{produces<MkFitEventOfHits>()},
+      useStripStripQualityDB_{iConfig.getParameter<bool>("useStripStripQualityDB")} {}
```

Comment:
Now that the `SiStripQuality` and `TrackerGeometry` are used only if `useStripStripQualityDB_ == true`, their consumes declaration should be changed to
```cpp
if (useStripStripQualityDB_) {
  qualityToken_ = esConsumes();
  geomToken_ = esConsumes();
}
```

Reply:
Done. Please, check.